glide.yaml

```diff
@@ -27,7 +27,7 @@ import:
 - package: github.com/evanphx/json-patch
   version: ba18e35c5c1b36ef6334cad706eb681153d2d379
 - package: k8s.io/client-go
-  version: v5.0.1
+  version: b044414c7d52baaf48bc86edb5c89bbd73524c0f
```

Comment:
What is "b044414c7d52baaf48bc86edb5c89bbd73524c0f". That looks like a random upstream git hash from a few days ago. The latest verified release of client-go is v6.0.0.

Reply:
@davidvossel I thought we work close to latest changes of k8s, it the reason why I set git hash, I will change it to a specific version. I think we will need to create `compatibility matrix` similar to one that `client-go` project has.